src/main/java/billgates/database/MySQLDatabaseGateway.java

```diff
@@ -471,6 +471,42 @@ INSERT INTO users (username, password, bill_id) VALUE (
         }
     }
 
+    @Override
+    public void deleteUser(QueryUserData user) {
```

Comment:
Do you not want to override a method for the ID and the Username? 

I think calling the method by passing an int or a username is easier than creating a `QueryUserData` class then throwing it in?

Reply:
Yes